src/core/request.js

```diff
@@ -60,6 +60,8 @@ const createError = (response, fallbackMessage, httpCode, httpSent) => {
     let err;
     if (response && response.errorCode) {
         err = new Error(`${response.errorCode} : ${response.errorMsg || response.errorMessage || response.error}`);
+    } else if (response && response.code) {
+        err = new Error(`${response.code} : ${response.message}`);
```

Comment:
Do we need to show code? Can you check with UX?

Reply:
Its like the version from L:62 so I didn't change anything, and judging that its JS object so it need a key value pair. Will need to change much more than this simple if to mitigate this problem.

Reply:
I've updated the code so no more dilemma about error code :)